scripts/setup-env.sh

```diff
@@ -127,3 +127,6 @@ echo ""
 alias python=python3
 
 export HDMI2USB_ENV=1
+
+# Set prompt
+PS1="(HDMI2USB)$PS1"
```

Comment:
Well, the prompt should probably evaluate them every time. I'll merge this now and we can make it better later.